### PR TITLE
Update Arc view display message style to support codes and numbering …

### PR DESCRIPTION
**Pull request by dharam1291**
Given below are the enhancement support by this PR

1. Add support of multiline when the user enters queries 
    Reason: Improve readability to relook at the prompt before sending it to LLM
2. Bot Display content:
    Now, the bot can display code in the code block and support numbering, indentation .. etc properly before that it displayed some funny character   
    @patwlan: currently used hardcode background box colour for code if this looks code then I'll move to theme :) 
    
   FYI: See the screenshot 
SC_1
SC_3
SC_4

### CODE REVIEW

File: lib/src/chat/chat_field.dart [hunk 1]
```diff
@@ -14,10 +14,13 @@ class ChatField extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return TextField(
+    return SingleChildScrollView(
```
Comment (patwlan) [inline lib/src/chat/chat_field.dart:17]: You should be able to remove SingleChildScrollView. Or is it required for some reason?

File: lib/src/chat/chat_field.dart [hunk 2]
```diff
@@ -14,10 +14,13 @@ class ChatField extends StatelessWidget {
...
+        child: TextField(
       controller: controller,
       onSubmitted: onSubmitted,
       decoration: const InputDecoration(),
-    );
+      keyboardType: TextInputType.multiline,
+      maxLines: 2,
```
Comment (patwlan) [inline lib/src/chat/chat_field.dart:23]: You can also set minLines to 1.

**Comment by patwlan** [comment]
This is looking good. If you rebase, we should be able to merge soon.

File: lib/src/chat/chat_field.dart (hunk 2 above)
Comment (dharam1291) [reply lib/src/chat/chat_field.dart:23]: for multiline support maxLines>1

**Pull request closed without merging**